warp/src/js_exports/raygun.rs

```diff
@@ -301,6 +304,137 @@ impl RayGunBox {
     }
 }
 
+/// impl RayGunGroup trait
+#[wasm_bindgen]
+impl RayGunBox {
+    //Update conversation name Note: This will only update the group conversation name
+    pub async fn update_conversation_name(
+        &mut self,
+        conversation_id: String,
+        name: String,
+    ) -> Result<(), JsError> {
+        self.inner
+            .update_conversation_name(Uuid::from_str(&conversation_id).unwrap(), &name)
+            .await
+            .map_err(|e| e.into())
+    }
+
+    /// Add a recipient to the conversation
+    pub async fn add_recipient(
+        &mut self,
+        conversation_id: String,
+        recipient: String,
+    ) -> Result<(), JsError> {
+        self.inner
+            .add_recipient(
+                Uuid::from_str(&conversation_id).unwrap(),
+                &DID::from_str(&recipient).unwrap_or_default(),
```

Comment:
You would want to avoid using `unwrap_or_default` here as if the key is invalid, it would generate a new key which would not be targeted towards the specific identity. 
```suggestion
                &DID::from_str(&recipient).unwrap(),
```